### ISSUE TITLE
Plot title truncated

### ISSUE THREAD

**Issue by cdhf** [ma-laforge/InspectDR.jl#25]
Plotting with julia 1.6.0, Plots 1.11.1 and InspectDR 0.4.0. When setting a plot title, the upper half of it is truncated, both in the interactive window and in exported files.

using Plots, InspectDR
inspectdr()
plot([1,2,3]; title="Test")

results in 

![screenshot](https://user-images.githubusercontent.com/1413583/112808595-7008ce80-9079-11eb-88f5-cba39fcfbaee.png)

**Comment by ma-laforge** [comment]
Yes. I don't increase area when font size increases. I will have to fix that in Plots.jl but it might take some time.

### Workaround for now
You will have to manually fix the font size in InspectDR:

```
using Plots, InspectDR
inspectdr()

p=plot([1,2,3]; title="Test")
pgui = display(p) #Display plot manually, and get ref to InspectDR object.

#Increase headroom for title:
layout = pgui.subplots[1].src.layout
layout[:valloc_top] *= layout[:font_title]._size/14
```
